core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -524,6 +530,7 @@ service FileSystemMasterClientService {
    */
   rpc ListStatus(ListStatusPRequest) returns (stream ListStatusPResponse);
 
+  rpc ListStatusCli(ListStatusCliPRequest) returns (ListStatusCliPResponse);
```

Comment:
for this PR, let's just focus on `loadMetadata`, and introduce only

```
rpc LoadMetadata(LoadMetadataPRequest) returns (LoadMetadataPResponse);
```

Let's not aim for multiple purposes in a single PR

Reply:
You ever suggested RPCs which associated with ListStatus should be merged into a common RPC, and this design is not complicated. So shaw I apply this design in another PR？